src/dmd/mars.d

```diff
@@ -1369,8 +1366,7 @@ extern(C) void flushMixins()
 
     assert(global.params.mixinFile);
     auto f = File(global.params.mixinFile);
-    OutBuffer* ob = global.params.mixinOut;
-    f.setbuffer(cast(void*)ob.data, ob.offset);
+    f.hijackBufferFrom(*global.params.mixinOut);
```

Comment:
What I don't like about this is the data is flowing from right to left. Data should flow from left to right, the whole "pipeline" thing.